drivers/SmartThings/matter-switch/profiles/light-level-colorTemperature-2700K-6500K.yml

```diff
@@ -0,0 +1,24 @@
+name: light-level-colorTemperature-2700K-6500K
+components:
+- id: main
+  capabilities:
+  - id: switch
+    version: 1
+  - id: switchLevel
+    version: 1
+    config:
+      values:
+        - key: "level.value"
+          range: [1, 100]
+  - id: colorTemperature
+    version: 1
+    config:
+      values:
+        - key: "colorTemperature.value"
+          range: [ 2700, 6500 ]
+  - id: firmwareUpdate
```

Comment:
Should there also be a capability for Color Control?

Reply:
If the device only supports color temperature, it does not require the color control capability (which is for cases where you can change the RGB color values rather than just color temp).